### PR TITLE
Convert gamble to slash command

### PR DESCRIPTION
**Pull request by tanndlin**

### CODE REVIEW

File: src/app/slash_plugins/gamble.plugin.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  description: 'Bet your Tacos on a coin flip',
+  options: [
+    {
+      name: 'amount',
+      description: 'The amount of Tacos to gamble',
+      type: 'STRING',
+      required: true,
+    },
+  ],
+  execute: async ({ interaction, container }) => {
```
Comment (suneettipirneni) [inline src/app/slash_plugins/gamble.plugin.ts:19]: Small nit, other plugins use the `async execute(…)` style.

File: src/app/slash_plugins/gamble.plugin.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      return;
+    }
+
+    await interaction.deferReply();
+
+    const userDoc = await container.pointService.getUserPointDoc(interaction.user.id);
+    const pointsToGamble = arg.toLowerCase() === 'all' ? userDoc.numPoints : parseInt(arg);
+
+    if (userDoc.numPoints === 0) {
+      await interaction.followUp('You have no Tacos to gamble!');
```
Comment (suneettipirneni) [inline src/app/slash_plugins/gamble.plugin.ts:32]:
```suggestion
      await interaction.followUp({ content: 'You have no Tacos to gamble!', ephemeral: true });
```

**Review by suneettipirneni — CHANGES_REQUESTED** [review]

File: src/app/slash_plugins/gamble.plugin.ts (hunk 1 above)
Comment (tanndlin) [reply src/app/slash_plugins/gamble.plugin.ts:19]: oh ok yeah i want to stick to the same pattern. just didnt realize you could do that

File: src/app/slash_plugins/gamble.plugin.ts (hunk 2 above)
Comment (tanndlin) [reply src/app/slash_plugins/gamble.plugin.ts:32]: I need to remember that is an option lol

**Review by suneettipirneni — APPROVED** [review]

**Pull request merged**